pyanaconda/modules/common/structures/storage.py

```diff
@@ -453,3 +453,61 @@ def get_root_device(self):
         :return: a device name or None
         """
         return self.mount_points.get("/")
+
+
+class PlatformMountPointData(DBusData):
+    """Platform required mount points data"""
```

Comment:
- Maybe add the Required to the name? I think this naming and docstring does not do a good job of explaining that the data is about something that is hypothetical and needs to be created, instead of something already existing.
- I wonder why have platform there? Is this related to some platform module, object, class,...?